tests/end2end/utils.py

```diff
@@ -632,6 +662,25 @@ def _skipif_api_feature_not_on_cpc_and_hmc(feature, cpc):
         skip_warn(f"API feature {feature} not available on HMC {console.name}")
 
 
+def skipif_no_partition_link_feature(cpc):
+    """
+    Skip the test if the "dpm-smcd-partition-link-management" API feature is
+    not enabled for the specified CPC, or if the CPC does not yet support it.
+
+    Note that the three partition-link related API features are always all
+    enabled or all disabled:
```

Comment:
Not fully correct. There are code levels for z16 where only SMC-D is enabled, but not HiSo and CTC.

Reply:
Changed the function to check that all PL related features are enabled. 
DONE.